src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -29,12 +29,13 @@ class FilesystemLoader extends \Twig_Loader_Filesystem
     /**
      * Constructor.
      *
-     * @param FileLocatorInterface        $locator A FileLocatorInterface instance
-     * @param TemplateNameParserInterface $parser  A TemplateNameParserInterface instance
+     * @param FileLocatorInterface        $locator  A FileLocatorInterface instance
+     * @param TemplateNameParserInterface $parser   A TemplateNameParserInterface instance
+     * @param string                      $rootPath The project's root dir
```

Comment:
Should use the same as https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Loader/Filesystem.php#L30